package.json

```diff
@@ -8,7 +8,10 @@
     "lint": "vue-cli-service lint"
   },
   "dependencies": {
+    "bootstrap-vue": "^2.21.2",
+    "bulma": "^0.9.2",
```

Comment:
It'd be nice if we could reduce the number of dependencies. If they are needed that's okay, but maybe we could talk about why?

Reply:
I'm not using those dependencies, in the future should I uninstall any dependencies that I'm not using?